GrowthViz-adults.py

```diff
@@ -25,10 +25,11 @@
 # 
 # Jupyter Notebooks have documentation cells, such as this one, and code cells like the one below. The notebook server can runs the code and provides results (if applicable) back in the notebook. The following code cell loads the libraries necessary for the tool to work. If you would like to incorporate other Python libraries to assist in data exploration, they can be added here. Removing libraries from this cell will very likely break the tool.
 
-# In[1]:
+# In[8]:
 
 
 from __future__ import print_function
```

Comment:
I *think* this can be removed. This was a convention during python 2 days to prepare for python 3's new print statement, which requires parentheses.

Reply:
Whoops, same import is still present in -pediatrics.